examples/src/main/java/com/vesoft/nebula/examples/GraphClientExample.java

```diff
@@ -149,6 +150,26 @@ public static void main(String[] args) {
                                                         value.columns.get(1).getInteger(),
                                                         new String(value.columns.get(2).getStr())));
             }
+
+            // Batch insert some edges for ScanEdgeInSpaceExample and SparkExample
+            for (long src = 1; src <= 10; src++) {
```

Comment:
Maybe we should have a test dataset. It's used by all the test case.

Reply:
fixed